### PR TITLE
Railroad diagrams.

### PR DESCRIPTION
**Pull request by walter-weinmann**
I've converted the parse file into 5 ebnf syntax files:

- **SQL 1 sql_list**: contains the sql_list clause
- **SQL 2 cursor_def**: contains the cursor_def clause and all its subclauses
- ...

In each file you'll find the reasoning lines of the parse file as a comment before the ebnf syntax. The files can be directly used as input grammar files in the Railroad Diagram Generator (http://bottlecaps.de/rr/ui).

The generated HTML file and the image files can then be downloaded.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ ebin/
 deps/
 src/sql_lex.erl
 src/sqlparse.erl
+/.idea
+/sqlparse.iml
```
Comment (c-bik) [inline .gitignore]: See https://github.com/K2InformaticsGmbH/sqlparse/pull/19#issuecomment-186172273

**Comment by c-bik** [comment]
@walter-weinmann
* Please move RailRoadDiagram and docs from `priv` to `docs`
* Please remove IDEA from `.gitignore`

**Comment by walter-weinmann** [comment]
Done - but I'm still testing and fine-tuning.

**Comment by c-bik** [comment]
Ok. Please re-open (this or a new PR) when you want me merge it to master.

**Comment by walter-weinmann** [comment]
First version finished - a couple of topics to be clarified next week.

**Comment by c-bik** [comment]
@walter-weinmann This is a closed PR. If you like me to merge, please re-open.

**Comment by walter-weinmann** [comment]
Thanks Bikram - this is all included in #27. So no need to reopen.

**Pull request closed without merging**